Neos.ContentRepository.Core/Classes/SharedModel/Exception/RootNodeAggregateTypeIsAlreadyOccupied.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Neos\ContentRepository\Core\SharedModel\Exception;
+
+use Neos\ContentRepository\Core\NodeType\NodeTypeName;
+
+/**
+ * @api Userland code might have to react to this
+ */
+final class RootNodeAggregateTypeIsAlreadyOccupied extends \DomainException
```

Comment:
```suggestion
final class RootNodeAggregateTypeIsAlreadyInUse extends \RuntimeException
```

Two changes:

- `*Occupied` -> `*InUse` (as per my earlier suggestion to change the naming slightly to better match the semantics)
- `\DomainException` -> `\RuntimeException` (funny, here I think `\DomainException` is indeed the wrong base class, since the root node type being already in use is a condition that can only happen at runtime)

Beyond this, I have a question:

Why is this exception speaking of a `RootNodeAggregateType` instead of a `RootNodeType`? Are (root) node aggregate types a concept that exists?